lib/hushed/date_service.rb

```diff
@@ -0,0 +1,55 @@
+module Hushed
+  class DateService
+
+    def build_date(date_string)
+      month = date_string.slice(0..1).to_i
+      day = date_string.slice(2..3).to_i
+      year = ("20" + date_string.slice(4..5)).to_i
+
+      Date.new(year, month, day)
+    end
+
+    def most_recent_day(date_array)
+      quick_sort(date_array).first
+    end
+
+    def quick_sort(date_array)
```

Comment:
i believe i initially tried using one of the sort methods that come with Ruby but i couldn't get any of them to work and just build a quick sort instead. also just tried again and i'm continuing to get failures so i think i'll leave this in unless you think it's significantly less efficient than using one of the provided sort methods